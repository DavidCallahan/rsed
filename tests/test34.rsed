$in = "1,2,3,4,5,6,7,8,9,10,11,,13"
split $in with ","
print "2=" $1
print <<END
1=$0
9=$8
13=$12
END
