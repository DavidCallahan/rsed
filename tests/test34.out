2=2
1=1
9=9
13=13
